OptimizelySDK/ProjectConfig.cs

```diff
@@ -293,6 +299,18 @@ private void Initialize()
                     }
                 }
             }
+
+            // Adding experiments in experiment-feature map.
+            foreach (var feature in FeatureFlags)
+            {
+                foreach (var experimentId in feature.ExperimentIds ?? new List<string>())
+                {
+                    if (ExperimentFeatureMap.ContainsKey(experimentId))
+                        ExperimentFeatureMap[experimentId].Add(feature.Id);
+                    else
+                        ExperimentFeatureMap[experimentId] = new List<string> { experimentId };
```

Comment:
Shouldn't we be having `feature.Id` and not `experimentId`?

Reply:
Oops, yeah that's a mistake. Fixed now.